src/main/java/com/minecolonies/coremod/colony/workorders/WorkOrderPlantationField.java

```diff
@@ -84,7 +84,7 @@ public void onAdded(final IColony colony, final boolean readingFromNbt)
         super.onAdded(colony, readingFromNbt);
         if (!readingFromNbt && colony != null && colony.getWorld() != null)
         {
-            ConstructionTapeHelper.placeConstructionTape(this, colony.getWorld());
+            ConstructionTapeHelper.placeConstructionTape(this, colony.getWorld(), colony);
```

Comment:
Is the level argument still necessary if we pass the whole colony anyway? (Applies to WorkOrderBuilding/WorkOrderDecoration too)